scss/_header.scss

```diff
@@ -0,0 +1,81 @@
+// Header styles
+header {
+    background-color: $brand-color;
+    height: auto;
+    
+    //Header content layout
+    .header-content {
+        @include flex-layout-row;
+        max-width: $wrapper-max-width;
+        @include margin-center;
+        padding: 0 $padding;
+    }
+    //Logo style
+    h1 {
+        color: $text-color-light;
+        font-size: 2.5em;
+        margin: 0;
+        padding: $padding/2;
+    }
+
+    //nav styles
+    nav {
+        // locally scoped variable
+        height: auto;
+        display: flex;
+        align-items: right;
+        align-self: center;
+    
+        ul {
+            //Help to show nav list on bigger than tablet screens
+            //@include flatten;
+            list-style: none;
+            display: flex;
+    
+                //Shows nav list on tablet size screens and smaller
+                @media (max-width: $tabletPortrait) {
```

Comment:
are we doing desktop first design?

Reply:
Let start with mobile. I just grabbed piece of my other code for nav and didn't modified it here. As you can see - navigation does not work properly yet because I did not finished it. Would you like to see completely finished parts of pages only? Or should I submit any progress?

Reply:
I think it's fine to submit any progress as long as it is working. Like if there is code for some functionality that is broken, it's better to just commit it. Once it's working, we can push it. This will help keep code is master clean.